cpp/tests/interop/from_arrow_test.cpp

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+#include <tests/interop/arrow_utils.hpp>
```

Comment:
What is our heuristic for whether files in `tests/` should use local includes? It seems like we might have inconsistency here but I didn't check that closely.

Reply:
Please file an issue.